cli/tests/inline_js_source_map_with_contents_from_graph.js.out

```diff
@@ -0,0 +1,4 @@
+error: Uncaught Error: Hello world!
+// throw new Error("Hello world!" as string);
```

Comment:
This looks fishy to me, why is there error thrown when the line is commented out?

Reply:
The error is not thrown by that file. The error is thrown by `cli/tests/inline_js_source_map_with_contents_from_graph.js`, which source maps to `cli/tests/inline_js_source_map_with_contents_from_graph.ts`.

Reply:
Sorry I don't understand, there is no `inline_js_source_map_with_contents_from_graph.ts` file here

Reply:
Sorry I mean't `cli/tests/inline_js_source_map.ts`.

Reply:
Okay, then again why is it throwing an error if that line is commented out :D ?

Reply:
Because that is not where the error is thrown. It is thrown at `throw new Error("Hello world!");` in `cli/tests/inline_js_source_map_with_contents_from_graph.js`, which source maps to `// throw new Error("Hello world!" as string);` in `cli/tests/inline_js_source_map.ts`, so this is what is displayed in the error. These tests use some hand crafted source maps to check that we get the source_line from the correct file. No compiler would ever emit anything like this.

Reply:
I see, maybe add some comments about hand crafted files and source maps? It's pretty convoluted and I didn't understand what was going on, might be helpful in the future